### ISSUE TITLE
[DataCap Application] <Dazhu County Wannongda agricultural development limited liability company> - <agriculture>

### ISSUE THREAD

**Issue by jauea** [NDLABS-Leo/Allocator-Pathway-ND-CLOUD#30]
### Version

1

### DataCap Applicant

Dazhu County Wannongda agricultural development limited liability company

### Project ID

1

### Data Owner Name

Dazhu County Wannongda agricultural development limited liability company

### Data Owner Country/Region

China

### Data Owner Industry

Resources, Agriculture & Fisheries

### Website

www.wannongda.net

### Social Media Handle

www.wannongda.net

### Social Media Type

Other

### What is your role related to the dataset

Dataset Owner

### Total amount of DataCap being requested

5PiB

### Expected size of single dataset (one copy)

1.2PiB

### Number of replicas to store

6

### Weekly allocation of DataCap requested

2PiB

### On-chain address for first allocation

f1inhnarm6mgotgmqqiiuff4l644egdyl3ktsvdei

### Data Type of Application

Slingshot

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
General projects: spice crop cultivation; Fruit cultivation; Tea planting; Vegetable cultivation; Nut cultivation; Bean planting; Tobacco cultivation; Sugar crop cultivation; Tree planting and management; Fertilizer sales; Agricultural machinery sales; Agricultural film sales; Conference and exhibition services; Marketing of pesticides for health use; Livestock and fishery feed sales; Sales of agricultural and sideline products; Sales of forestry products; Primary processing of edible agricultural products; Wholesale of edible agricultural products; Retail of edible agricultural products; Fresh vegetable wholesale; Purchase of primary agricultural products; Sales of specialized machinery for agriculture, forestry, animal husbandry, auxiliary and fishery; Internet sales (except sales of goods that require a license); Installation and maintenance of specialized machinery for agriculture, forestry, animal husbandry and secondary fishery; Crop pest control services; Business agency service; Technical service, technology development, technology consultation, technology exchange, technology transfer, technology.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Various farming techniques.
```

### Where was the data currently stored in this dataset sourced from

My Own Storage Infra

### If you are a data preparer. What is your location (Country/Region)

China

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
I'm a data preparer and I use Lotus/Boost
```

### If you are not preparing the data, who will prepare the data?  (Provide name and business)

```text
myself
```

### Has this dataset been stored on the Filecoin network before? If so, please explain and make the case why you would like to store this dataset again to the network. Provide details on preparation and/or SP distribution.

```text
no
```

### Please share a sample of the data

```text
https://cloud.189.cn/web/share?code=IZbmqaJ7zaia
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

Permanently

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, South America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives

### How did you find your storage providers

Slack, Partners

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f02822222 Hong Kong
f02368946 Beijing
f02362412 California
f02639492 California
f03166666 Hongkong
f03166668 Hongkong
```

### How do you plan to make deals to your storage providers

Boost client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by jauea** [comment]
https://cloud.189.cn/web/share?code=nEriQzzIFn2i（访问码：cih5）

**Comment by jauea** [comment]
@NDLABS-Leo Reshared and updated password

**Comment by NDLABS-Leo** [comment]
@jauea 
Hello, thank you for your work. I noticed that the shared videos are primarily agriculture-based video data.

1.	Can you provide any evidence to confirm that your application is a corporate submission?
2.	Do you actually have enough data to meet the weekly application volume of 2 PiB?
3.	Please confirm the SP nodes for the first batch of collaboration, as I need to review them.

**Comment by jauea** [comment]
> Can you provide any evidence to confirm that your application is a corporate submission?

I can send you an email confirmation using our company's domain email address

>Do you actually have enough data to meet the weekly application volume of 2 PiB?

It may not be fulfilling at the moment, I'll amend it to 1PiB per week

>Please confirm the SP nodes for the first batch of collaboration, as I need to review them.

Updated

**Comment by NDLABS-Leo** [comment]
@jauea 
Alright, you can send it to my email.[[[xiazhe@ndlabs.io](mailto:xiazhe@ndlabs.io)]
Please confirm the SP nodes for the first batch of collaboration, as I need to review them.

**Comment by jauea** [comment]
Mail sent

已再次确认前期将合作的sp

**Comment by NDLABS-Leo** [comment]
Current list of received SP nodes：
f03226666  HK
f02819999  US
f03166677 Guangzhou
f03166688 Guangzhou

**Comment by NDLABS-Leo** [comment]
f03226666  HK - New nodes
f02819999  US - New nodes

<img width="706" alt="image" src="https://github.com/user-attachments/assets/b62615df-f89c-4dc5-9cea-7c2c3b83f11c">
<img width="279" alt="image" src="https://github.com/user-attachments/assets/38e8d97b-e38d-4530-9943-de2e6898d66c">

<img width="697" alt="image" src="https://github.com/user-attachments/assets/815a00bb-762a-4f4f-8e21-4a9c12385706">
<img width="309" alt="image" src="https://github.com/user-attachments/assets/7c673b41-9acb-42f5-aa5f-0758ea4dc5da">

**Comment by NDLABS-Leo** [comment]
<img width="732" alt="image" src="https://github.com/user-attachments/assets/b1a9aa88-a76b-40f7-aa95-0c668038b5a6">

Confirmed receipt of an email from the domain “wannongda.net.”

**Comment by NDLABS-Leo** [comment]
@jauea We will release a small amount of datacap to initiate the project. Please ensure proper node retrieval, data backups, and the geographic distribution of backup nodes. The project will only proceed if it fully complies with the rules of both Filecoin Plus (Fil+) and our Allocator.

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
@jauea 
The report aligns with the disclosed nodes, with a retrieval rate around 70%. Please maintain this level.

For the upcoming rounds, new nodes should be added, and thorough data backups are essential. Could you confirm if your team is prepared for these requirements?

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/e28f45b9-0010-43bb-bf61-39dec7b5a2c9">

**Comment by jauea** [comment]
YES, we are always looking for new sp partnerships and will be adding more in the future!

**Comment by NDLABS-Leo** [comment]
f03226666  HK 
f02819999  US

@jauea Will these two nodes be included in the next round?

**Comment by jauea** [comment]
We contacted the new sps
 Will these two nodes be included in the next round
 f03175111 HK
f03175168 japan

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by jauea** [comment]
checker:manualTrigger

**Comment by NDLABS-Leo** [comment]
If there are any new developments in the project, please feel free to contact me at any time.

**Comment by jauea** [comment]
Due to the long time period and changes in some sp investment plans, I have changed the application form and added new SPs, please confirm

**Comment by NDLABS-Leo** [comment]
@jauea 
Hello, I noticed that the files you submitted seem to overlap with those in applications to other allocators. I recommend submitting your application through only one allocator.
I will proceed to close this application.
